sql/step_a10X_create_table_inv_and_invr.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE OH_MEDICALDSRINVENTORY (
+	MINVT_ID int NOT NULL AUTO_INCREMENT ,
+	MINVT_STATUS varchar (10)  NOT NULL ,
+	MDSR_DESC varchar (100)  NOT NULL ,
+	MINVT_DATE datetime NOT NULL ,
+	MINVT_US_ID_A varchar (50) NOT NULL,
+	MINVT_REFERENCE varchar (50) NOT NULL,
+	MINVT_TYPE varchar(30) NOT NULL,
+	MINVT_WRD_ID_A varchar(1) NULL DEFAULT NULL,
+	MINVT_LOCK int NOT NULL default 0,
+	MINVT_CREATED_BY VARCHAR(50) NULL DEFAULT NULL,
+  	MINVT_CREATED_DATE datetime NULL DEFAULT NULL,
+  	MINVT_LAST_MODIFIED_BY VARCHAR(50) NULL DEFAULT NULL,
+  	MINVT_LAST_MODIFIED_DATE datetime NULL DEFAULT NULL,
+  	MINVT_ACTIVE TINYINT(1) NOT NULL DEFAULT 1,
+	PRIMARY KEY (MINVT_ID ) ,
+	FOREIGN KEY (MINVT_US_ID_A) REFERENCES OH_USER (US_ID_A) ,
+	FOREIGN KEY (MINVT_WRD_ID_A) REFERENCES OH_WARD (WRD_ID_A)
+) ENGINE = INNODB DEFAULT CHARACTER SET utf8;
+
+CREATE TABLE OH_MEDICALDSRINVENTORYROW (
+	MINVTR_ID int NOT NULL AUTO_INCREMENT ,
+	MINVTR_THEORETIC_QTY float  NOT NULL default 0 ,
+	MINVTR_REAL_QTY float  NOT NULL default 0 ,
+	MINVTR_U_PRICE float  NOT NULL default 0 ,
```

Comment:
This looks wrong, the COST (not the price) is linked to the Lot, that is already referenced with MINVTR_LT_ID_A

Reply:
we have a certain product that does not have a lot but we would like to have the last purchase price. this column allows us to obtain the last purchase price of the item. of course if this item has a lot we can refer to the price of the lot.

Reply:
this sounds like a potential bug: for a while now we have lot information also at ward level, so every product must have a lot, otherwise it could lead to inconsistencies.

Note: from the hospital point of view the product is a "cost" and we should refer to its price a such. 
for the hospital a product has a "price" when it is sold to the patients (via pricelists)

Reply:
but again: since you cannot have a product without lot, that value should be taken from the referenced lot, or am I wrong?

Reply:
no okay so we can remove it